jans-auth-server/pom.xml

```diff
@@ -17,17 +17,24 @@
 		<jans-notify.version>1.0.0-SNAPSHOT</jans-notify.version>
 		<jans-fido2.version>1.0.0-SNAPSHOT</jans-fido2.version>
 
+		<servlet.version>5.0.0</servlet.version>
+
 		<activemq.version>5.15.14</activemq.version>
 
 		<jettison.version>1.4.1</jettison.version>
 
 		<slf4j.version>1.7.25</slf4j.version>
 
-		<omnifaces.version>2.6.9</omnifaces.version>
+		<!-- <omnifaces.version>2.6.9</omnifaces.version> -->
```

Comment:
We need to check if we not use this library in all xhtml (including custom authenticators). if we use it we need to use pure JSF